### ISSUE TITLE
Adding the Barcode functionality

### ISSUE THREAD

**Issue by kartikparmar** [TycheSoftwares/bkap-printable-tickets#35]
Along with the ticket information that is being sent to the customer, we should also show add the barcode to it.
Screenshot - https://prnt.sc/jsSa1dvIzQez

There should be some options in the plugin to create how they want to create the Barcode/QR-Code. 

Options could be as follows
Barcode Type 
Barcode Color
https://prnt.sc/dp2GKiliRrtK

Barcode Types could be as shown in this screenshot - https://prnt.sc/UyM6650CR7VQ

**Comment by ashokrane** [comment]
@kartikparmar Is this a client issue?

**Comment by kartikparmar** [comment]
@ashokrane No. But in the past, we had one client for whom we added a barcode in the ticket email.

**Comment by ashokrane** [comment]
Got it. Unless multiple clients come back & ask for this, we shouldn't work on this. We should close it open if this was asked more than 2 years ago by the client.

**Comment by komal-maru** [comment]
One client is asking for this functionality: https://support.tychesoftwares.com/conversation/287?folder_id=4#thread-928

**Comment by komal-maru** [comment]
one more client asked for this: https://support.tychesoftwares.com/conversation/19766?folder_id=13#thread-151940
